<p>Don't define a subclass of java.lang.Throwable which is not a subclass of java.lang.Exception</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO 
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO 
</pre>
